products/chromeos.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Chrome OS
+category: os
+tags: linux-distribution
+iconSlug: googlechrome
+permalink: /chromeos
+alternate_urls:
+-   /chrome-os
+versionCommand: cat /etc/issue
+releasePolicyLink: https://chromeos.dev/en/education/chromeos-lts
+changelogTemplate: https://support.google.com/chrome/a/answer/10314655#__RELEASE_CYCLE__
+releaseColumn: false
+activeSupportColumn: false
+releaseDateColumn: true
+
+releases:
+
+-   releaseCycle: "116"
+    releaseDate: 2023-08-22
+    eol: false
+
+-   releaseCycle: "115"
+    releaseDate: 2023-07-12
+    eol: 2023-08-22
+
+-   releaseCycle: "114"
+    releaseDate: 2023-05-24
+    eol: 2023-07-12
+
+-   releaseCycle: "113"
+    releaseDate: 2023-04-26
+    eol: 2023-05-24
+
+-   releaseCycle: "112"
+    releaseDate: 2023-03-29
+    eol: 2023-04-26
+
+-   releaseCycle: "111"
+    releaseDate: 2023-03-01
+    eol: 2023-03-29
+
+-   releaseCycle: "110"
+    releaseDate: 2023-02-01
+    eol: 2023-03-01
+
+-   releaseCycle: "109"
+    releaseDate: 2023-01-10
+    eol: 2023-02-01
+
+-   releaseCycle: "108"
```

Comment:
[LTS releases](https://chromeos.dev/en/education/chromeos-lts) are not marked as such.

Reply:
LTS releases still need to be marked as such.